backend/usb-darwin.c

```diff
@@ -1429,32 +1429,34 @@ static kern_return_t load_classdriver(CFStringRef	    driverPath,
                                             _CUPS_FILE_CHECK_DIRECTORY, 1,
                                             Iterating ? NULL : _cupsFileCheckFilter, NULL);
 
-  if (result && driverPath)
-    return (load_classdriver(NULL, interface, printerDriver));
-  else if (result)
+  if (result)
```

Comment:
Unnecessary change, please limit macOS changes to bug fixes only (Apple isn't actually using OpenPrinting CUPS sources)